### PR TITLE
replaced ==== with ===

### PR DESCRIPTION
**Pull request by awslabspl**

### CODE REVIEW

**Comment by eliykat** [comment]
Hi @awslabspl, if you would like us to consider your PR for merging, please provide an explanation of your changes. It's unclear from the diff exactly what this does. You'll also need to sign the CLA.

**Comment by awslabspl** [comment]
@eliykat CLA in OSS project? Are you OK?

File: src/content/autofill.js [hunk 1]
```diff
@@ -42,12 +42,12 @@
 
     function collect(document, undefined) {
         // START MODIFICATION
-        var isFirefox = navigator.userAgent.indexOf('Firefox') !== -1 || navigator.userAgent.indexOf('Gecko/') !== -1;
+        var isFirefox = navigator.userAgent.indexOf('Firefox') === -1 || navigator.userAgent.indexOf('Gecko/') === -1;
```
Comment (Code-DJ) [inline src/content/autofill.js:45]: This will return false on Firefox. Is this intentional?

Comment (awslabspl) [reply src/content/autofill.js:45]: of course. FF support should be phased out from more & more software. FF is no longer valid browser.

**Pull request closed without merging**